src/cyberdropDownloader/cyberdropDownloader.csproj

```diff
@@ -57,17 +57,23 @@
     <ManifestKeyFile>cyberdropDownloader_TemporaryKey.pfx</ManifestKeyFile>
   </PropertyGroup>
   <PropertyGroup>
-    <GenerateManifests>true</GenerateManifests>
+    <GenerateManifests>false</GenerateManifests>
   </PropertyGroup>
   <PropertyGroup>
-    <SignManifests>true</SignManifests>
+    <SignManifests>false</SignManifests>
   </PropertyGroup>
   <PropertyGroup>
     <ApplicationIcon>duck.ico</ApplicationIcon>
   </PropertyGroup>
   <PropertyGroup>
     <SignAssembly>false</SignAssembly>
   </PropertyGroup>
+  <PropertyGroup>
+    <TargetZone>LocalIntranet</TargetZone>
```

Comment:
what this do?

Reply:
I didn't add that, though it most likely has to do with the fact i had to disable clickonce in order to buid

Reply:
> I didn't add that, though it most likely has to do with the fact i had to disable clickonce in order to buid

to add to this, I didn't touch source other than reorganizing the files. I did do a test build which required me disabling clickonce and removing the temp key, but other than that nothing else.